app/views/management/_container.html.erb

```diff
@@ -0,0 +1,36 @@
+
+<div class="container service-container">
+  <h3> Containers </h3>
+  <table class="table table-bordered table-striped">
+    <thead class="thead-dark">
+      <tr>
+        <th scope="col"> Name </th>
+        <th scope="col"> Image Tag </th>
+      </tr>
+    </thead>
+      <tr>
+        <td> Management </td>
+        <td> <%= "#{ENV['REGISTRY_HOST'] + ENV['REGISTRY_URI'] + ":" + ENV['MANAGEMENT_TAG']}" %> </td>
+      </tr>
+      <tr>
+        <td> DB </td>
+        <td> postgres </td>
+      </tr>
+      <tr>
+        <td> Blacklight </td>
+        <td> </td>
+      </tr>
+      <tr>
+        <td> Solr </td>
+        <td> <%= "yalelibraryit/dc-solr:" + "#{ENV['SOLR_TAG']}" %> </td>
+      </tr>
+      <tr>
+        <td> iiif_image </td>
```

Comment:
I think part of the difficulty of this ticket is that, currently, the deploy is being done using https://github.com/yalelibrary/yul-dc-camerata, and we'd want this page to show the versions that were in the latest Camerata deployment, not necessarily what is set in the management application itself.

Coordinating configuration between the applications is a tricky problem that is still in flux, but I see grappling with it as an integral part of this ticket. The fact that none of the non-Management tags are included suggests that maybe more thought could be put to that aspect of this problem.

Reply:
@maxkadel I'm not sure I understand what you mean here. This page reflects what is running in the container space at run time, not at build time.  Because the ENV vars are used, this page can simply report the available information from the running instance and therefor what is "up" at the given moment in time.  Camerata sets the correct info when it deploys to ECS or runs locally so this page should have what it needs when management is running alongside the other services. And when those services are not available, these other tags should be blank. If the names of those tags change then this code will need to change, but I don't really see a way around that.

Reply:
re: my comment about not grappling with the deployed environment was based on the fact that this partial is not yet attempting to show the Database, Blacklight, or IIIF image server versions. I agree that it will still work, although I think maybe Camerata is using a "SERVICE_VERSION" rather than tag (d'oh).